### PR TITLE
Fix/wc30 compatibility

### PR DESCRIPTION
**Pull request by bor0**

### CODE REVIEW

File: woocommerce-gateway-paypal-express-checkout.php [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?php
 /**
  * Plugin Name: WooCommerce PayPal Express Checkout Gateway
- * Plugin URI: https://www.woothemes.com/products/woocommerce-gateway-paypal-express-checkout/
+ * Plugin URI: https://woocommerce.com/products/woocommerce-gateway-paypal-express-checkout/
  * Description: A payment gateway for PayPal Express Checkout (https://www.paypal.com/us/webapps/mpp/express-checkout).
- * Version: 1.2.0
- * Author: Automattic
+ * Version: 1.2.1
```
Comment (gedex) [inline woocommerce-gateway-paypal-express-checkout.php]: 1.2.0 is not out from the oven. See list of releases in GH and latest version and https://wordpress.org/plugins/woocommerce-gateway-paypal-express-checkout/.

File: woocommerce-gateway-paypal-express-checkout.php [hunk 2]
```diff
@@ -36,7 +36,7 @@ function wc_gateway_ppec() {
 	if ( ! isset( $plugin ) ) {
 		require_once( 'includes/class-wc-gateway-ppec-plugin.php' );
 
-		$plugin = new WC_Gateway_PPEC_Plugin( __FILE__, '1.2.0' );
+		$plugin = new WC_Gateway_PPEC_Plugin( __FILE__, '1.2.1' );
```
Comment (gedex) [inline woocommerce-gateway-paypal-express-checkout.php]: same as above

File: readme.txt [hunk 3]
```diff
@@ -85,9 +85,12 @@ https://gist.github.com/mikejolley/ad2ecc286c9ad6cefbb7065ba6dfef48
 
 == Changelog ==
 
+= 1.2.1 =
```
Comment (gedex) [inline readme.txt]: same. Keep it under 1.2.0.

**Comment by bor0** [comment]
@gedex one more round please.

**Pull request merged**